### PR TITLE
Adopt Microsoft.VisualStudio.Threading

### PR DESCRIPTION
**Pull request by sharwell**
Related to (but not blocked by) gitextensions/gitextensions#4601

📝 If accepted, please do not rebase or squash this pull request during the merge.

### CODE REVIEW

File: NBug/NBug.csproj [hunk 1]
```diff
@@ -10,9 +10,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NBug</RootNamespace>
     <AssemblyName>NBug</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
Comment (gerhardol) [inline NBug/NBug.csproj]: 4.6.1 in GE

Comment (RussKie) [reply NBug/NBug.csproj]: I think it is safe to move it to the same version as the rest of the app.

Comment (sharwell) [reply NBug/NBug.csproj]: It should of course be possible, but nothing in this change required upgrading past 4.5. I would recommend implementing the unification as a separate change if it's desired.

**Comment by sharwell** [comment]
@gerhardol @RussKie Can we leave the unification for a separate PR? There are other repositories that are not unified on 4.6.1 and it seems outside the normal scope for this change.

**Comment by RussKie** [comment]
Yes, sure can

**Pull request merged**